src/io/flutter/view/FlutterView.java

```diff
@@ -632,16 +632,18 @@ public void perform(AnActionEvent event) {
 
   RepaintRainbowAction(@NotNull FlutterApp app) {
     super(app, "Show Repaint Rainbow", "Show Repaint Rainbow", FlutterIcons.RepaintRainbow);
-
     setExtensionCommand(SHOW_REPAINT_RAINBOW);
+    setEnabledText("Hide Repaint Rainbow");
+    setDisabledText("Show Repaint Rainbow");
   }
 }
 
 class ToggleInspectModeAction extends FlutterViewToggleableAction {
   ToggleInspectModeAction(@NotNull FlutterApp app) {
     super(app, "Toggle Select Widget Mode", "Toggle Select Widget Mode", AllIcons.General.LocateHover);
-
     setExtensionCommand("ext.flutter.inspector.show");
+    setEnabledText("Disable Select Widget Mode");
```

Comment:
Extend the constructor of FlutterViewToggleableAction to take a couple more parameters. That way it will be harder to forget to add these.

Reply:
Done.